### PR TITLE
Logger Update and Making Dev Less Painful

### PR DESCRIPTION
**Pull request by bcabio**
# Logger Update

Logger update implements a new module `log_handler.py` that has a new class `PBLogHandler` which extends the `logging.StreamHandler` class. This allows for a resuable log handler. With this new log handler, rather than having print statements in production, we'll have logger statements instead which can later be implemented to keep records i.e. logging to a file. 

# Making Dev Less Painful

Implemented a way to KeyboardInterrupt the websocket. Before, it was not possible and you'd have to close the terminal in order to get another instance running.

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -21,13 +21,25 @@
...
+
+logger = logging.getLogger('Bot')
+logger.setLevel(logging.INFO)
+logger.addHandler(log_handler.PBLogHandler())    
+
 class Bot(object):
-    admin_env_string = os.environ['PB_ADMIN']
-    ADMIN = admin_env_string.split(',')
+    # admin_env_string = os.environ['PB_ADMIN']
+    # ADMIN = admin_env_string.split(',')
```
Comment (mhijazi1) [inline bot.py]: You prolly commented this out for dev, but it should not be commented for prod

File: reactbot.py [hunk 2]
```diff
@@ -92,9 +101,12 @@ def on_message(self, ws, message):
     def on_message_thread(self, message):
         s = message.decode('utf-8')
         message_json = json.loads(unicode(s))
-        print "PantherBot:LOG:message:" + message_json["type"]
+        logger.info(message_json["type"].replace("_"," ").title())
         
         if "message" == message_json["type"]:
+            # if "text" in message_json and "bot_message" not in message_json["subtype"]  :
+            #     message_array = message_json["text"].split()
+            #     logger.info(message_json["type"].replace("_", " ").title() + ": " + message_array[0] + "... ")
```
Comment (mhijazi1) [inline reactbot.py]: Why is this here but commented?

File: webscraper.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
+from selenium import webdriver
```
Comment (mhijazi1) [inline webscraper.py]: Is this ment to be in here? Don't see any commits that mention it, seems kinda out of place for this PR

Comment (bcabio) [reply webscraper.py]: Oops. From previous dev stuff. I'll rm it.

File: reactbot.py (hunk 2 above)
Comment (bcabio) [reply reactbot.py]: Things I was testing. Lemme finish that and get back to you.

File: bot.py (hunk 1 above)
Comment (bcabio) [reply bot.py]: Kk

**Comment by bcabio** [comment]
_bump_

**Comment by bcabio** [comment]
Hold on don't merge this yet. There are some unforeseen errors in the `/scripts` files that I have to address.

**Review by ghost — APPROVED** [review]
